src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -79,225 +125,473 @@ const useStyles = makeStyles()(() => ({
+        initialValues={{
+          statusValue: statusValue,
+          pledgeCurrency: amountCurrency,
+          pledgeAmount: amount,
+          pledgeFrequency: frequencyValue,
+        }}
+        validationSchema={appealFormSchema}
+        onSubmit={async (values) => {
+          await onSubmit(values);
+        }}
+      >
+        {({
+          values: {
+            statusValue,
+            pledgeCurrency,
+            pledgeAmount,
+            pledgeFrequency,
+          },
+          handleSubmit,
+          setFieldValue,
+          errors,
+        }): ReactElement => {
+          const modalContact = {
+            id: id,
+            status: statusValue,
+            name: name,
+            pledgeCurrency,
+            pledgeAmount,
+            pledgeFrequency,
+          };
+
+          return (
+            <Form onSubmit={handleSubmit}>
+              <Grid container className={classes.formWrapper}>
+                <Card className={classes.formInner}>
+                  <Grid container>
+                    <Grid item lg={4} md={4} xs={12}>
+                      <Box
+                        display="flex"
+                        p={2}
+                        alignItems="center"
+                        className={classes.left}
+                      >
+                        <ContactAvatar
+                          src={avatar || ''}
+                          aria-label="Contact Avatar"
+                        />
+                        <Box display="flex" flexDirection="column" ml={2}>
+                          <Link
+                            data-testid="contactSelect"
+                            underline="hover"
+                            onClick={() => setContactFocus(id, 'Donations')}
+                          >
+                            <Typography variant="subtitle1">{name}</Typography>
+                          </Link>
+                          <Typography variant="subtitle2">
+                            {`Current: ${statusTitle} ${
+                              amount && amountCurrency
+                                ? currencyFormat(amount, amountCurrency, locale)
+                                : ''
+                            } ${frequencyTitle}`}
+                          </Typography>
+                        </Box>
+                      </Box>
+                    </Grid>
+                    <Grid item xs={12} md={8} lg={8} className={classes.right}>
+                      <Grid
+                        container
+                        style={{ paddingRight: theme.spacing(1) }}
+                      >
+                        <Grid item xs={12} md={6} lg={12}>
+                          <Box className={classes.boxTop}>
+                            <FormControl fullWidth>
+                              <InputLabel id="status-label">
+                                {t('Status')}
+                              </InputLabel>
+                              <Select
```

Comment:
For some reason, empty select's have their labels off center. Maybe there is a custom style affecting it?
<img width="773" alt="Screenshot 2024-08-08 at 5 50 13 PM" src="https://github.com/user-attachments/assets/74b39a06-63ab-47e6-a8f0-29559098306a">

Reply:
Also, now that I'm looking at this I think it would be best to remove "0" from the amount if there is no amount set.